starteam/starcellbio_html
# Implemented fixed and live for facs

#### What are the relevant tickets?

Fix #631 
#### What's this PR do?

Add support for two types of cell treatment in facs: 'live' and 'fixed'.
#### Where should the reviewer start?

assignments/scb_ex/assignment.js
I modified NSF assignment 2 structure to one that will be used by assignment builder, that supports 'Live' and 'Fixed' cell treatment options in Flow Cytometry. 
#### How should this be manually tested?

Work through Exercise 2 in the sample exercises. After you choose the technique (facs), on the Sample Prep page notice options given for 'Cell treatment', 'Analysis Type', 'Conditions'
